progress_bar.py

```diff
@@ -0,0 +1,21 @@
+# Print iterations progress
+def printProgressBar (iteration, total, prefix = '', suffix = '', decimals = 1, length = 100, fill = '█', printEnd = "\r"):
```

Comment:
stay consistent with function names - [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) suggests snake case

Reply:
Also, once you switch to typer, this function can probably just go away - https://typer.tiangolo.com/tutorial/progressbar/

Reply:
I renamed it according to the naming convention.